env-example

```diff
@@ -11,11 +11,17 @@ CMS_TTL=
 AUTH_HOST=id.staging.prx.tech
 # for metrics.prx.dev
 AUTH_CLIENT_ID=ea0e04f7c85a8d4826dcf2747126a53a95014f51
```

Comment:
Is this a new auth id for https://metrics.prx.dev? It gives me a CORS error, but I can use the one I was previously using `nz8hAjerWIW5rudwAWPPd1zC1Lawt89HMUNVJpBa` and my callback url is https, no warnings or errors.

Reply:
yeah, `ea0e04f7c85a8d4826dcf2747126a53a95014f51` is the staging key. `nz8hAjerWIW5rudwAWPPd1zC1Lawt89HMUNVJpBa` is the production key.